win32ss/user/user32/controls/appswitch.c

```diff
@@ -611,8 +613,6 @@ LRESULT WINAPI DoAppSwitch( WPARAM wParam, LPARAM lParam )
    if (switchdialog || Esc) return 0;
 
    hwndActive = GetActiveWindow();
```

Comment:
Since you removed the hwndActive test just below, and according to what the lParam if block does, it may be better to move this `hwndActive = GetActiveWindow();` line down after the if() block and just before the `hwndActive` check when you start the capture.